src/RAGKit.jl

```diff
@@ -1,10 +1,22 @@
 using HTTP, Gumbo, AbstractTrees, URIs
 using Gumbo: HTMLDocument, HTMLElement
 using EzXML
+using PromptingTools
+const PT = PromptingTools
+const RT = PromptingTools.Experimental.RAGTools
+using LinearAlgebra, Unicode, SparseArrays
+using HDF5
+using Tar
+using Inflate
+
+using SHA
+using Serialization, URIs
 # using Regex
 
 # using Robots
 
```

Comment:
Define your inclusions here. Make sure to couple any exported variables with the inclusions.

```julia
export some_function
include("my_functions.jl")
```